website/docs/r/mssql_database_vulnerability_assessment_rule_baseline.html.markdown

```diff
@@ -0,0 +1,131 @@
+  name                  = "accteststoragecontainer"
+  storage_account_name  = azurerm_storage_account.example.name
+  container_access_type = "private"
+}
+
+resource "azurerm_mssql_server_security_alert_policy" "example" {
+  resource_group_name = azurerm_resource_group.example.name
+  server_name         = azurerm_sql_server.example.name
+  state               = "Enabled"
+}
+
+resource "azurerm_sql_database" "example" {
+  name                = "mysqldatabase"
+  resource_group_name = azurerm_resource_group.example.name
+  server_name         = azurerm_sql_server.example.name
+  location            = azurerm_resource_group.example.location
+  edition             = "Standard"
+}
+
+resource "azurerm_mssql_server_vulnerability_assessment" "example" {
+  resource_group_name        = azurerm_resource_group.example.name
+  server_name                = azurerm_sql_server.example.name
+  storage_container_path     = "${azurerm_storage_account.example.primary_blob_endpoint}${azurerm_storage_container.example.name}/"
+  storage_account_access_key = azurerm_storage_account.example.primary_access_key
+
+  depends_on = [
+    azurerm_mssql_server_security_alert_policy.example
+  ]
+}
+
+resource "azurerm_mssql_database_vulnerability_assessment_rule_baseline" "example" {
+  resource_group_name = azurerm_resource_group.example.name
+  server_name         = azurerm_sql_server.example.name
+  database_name       = azurerm_sql_database.example.name
+  rule_id             = "VA2065"
+  baseline_name       = "master"
+  baseline_result {
+    result = [
+      "allowedip1",
+      "123.123.123.123",
+      "123.123.123.123"
+    ]
+  }
+  baseline_result {
+    result = [
+      "allowedip2",
+      "255.255.255.255",
+      "255.255.255.255"
+    ]
+  }
+
+  depends_on = [
+    azurerm_mssql_server_vulnerability_assessment.example
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `resource_group_name` - (Required) The name of the resource group that contains the MS SQL Server. Changing this forces a new resource to be created.
+
+* `server_name` - (Required) Specifies the name of the MS SQL Server. Changing this forces a new resource to be created.
+
+* `database_name` - (Required) Specifies the name of the MS SQL Database. Changing this forces a new resource to be created.
+
+* `rule_id` - (Required) The vulnerability assessment rule ID. Changing this forces a new resource to be created.
+    
+* `baseline_name` - (Optional) The name of the vulnerability assessment rule baseline. Valid options are `default` and `master`. `default` implies a baseline on a database level rule and `master` for server level rule. Defaults to `default`. Changing this forces a new resource to be created.
+
+* `baseline_result` - (Required) List of nested baseline result definitions. Each baseline result contains the following properties:
+
+  * `result` - (Required) A list representing a result of the baseline.
```

Comment:
```suggestion
* `result` - (Required) A list representing a result of the baseline.
```

Reply:
I put the documentation in a separate block documentation. Is that fine for you?

Reply:
👍